### ISSUE TITLE
[996580] Spyro Reignited Trilogy - Problem with pad and keyboard on Spyro 2

### ISSUE THREAD

**Issue by neur0mancer746** [ValveSoftware/Proton#3216]
# Compatibility Report
- Name of the game with compatibility issues: Spyro Reignited Trilogy, in particular Spyro 2
- Steam AppID of the game: 996580

## System Information
- GPU: AMD RX560
- Driver/LLVM version: Mesa 19.1.2 - padoka PPA
- Kernel version: 5.0.0-32-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/emmecii/167d67b4f801ccf992ee93e88e1208f9
- Proton version: 4.11-8

## I confirm:
- [X] that I haven't found an existing compatibility report for this game. _A compatibility report for this game already exist, but this new issue is particular and unrelated to Media Foundation issue_ 
- [X] that I have checked whether there are updates for my system available.

[steam-996580.log](https://github.com/ValveSoftware/Proton/files/3838547/steam-996580.log)

## Symptoms 
Gamepad on Spyro 2 doesn't properly recognize the Y button on X360 pad contextually to the hovering  action or sight-to-shoot action. The same function seems to be "transferred" pressing the right analog stick (but not for the hovering). Also the keyboard is affected, where spacebar and left mouse button doesn't work at all even if reassigned into the game
tl;dr https://www.youtube.com/watch?v=IHerojeRuaM that part can't be done for the Y button bug

I noticed this issue appear only when i load a new world level. At startup, there isn't any issue. Seems also I am the only that have this issue, so this is why i am opening a new issue on this game because is unrelated to a previous issue posting for Spyro 

At the moment i haven't found a level where hovering is necessary, but can be problematic

## Reproduction

Load a world level passing through a portal, then try to hovering after gliding with Y, or sight in first person perspective or when you aim to an enemy with Y / try to use flame with left button mouse or jump with spacebar

**Comment by kisak-valve** [comment]
Hello @emmecii, we're using one issue report per unofficially supported game title, so I've gone ahead and transferred this issue report to https://github.com/ValveSoftware/Proton/issues/3035#issuecomment-553156667.
